__tests__/app.test.js

```diff
@@ -183,6 +183,50 @@ describe("nc-games app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id/comments", () => {
+    it("returns an array of comments for the given review_id under the key of 'comments'", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments).toHaveLength(3);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
```

Comment:
```js
              comment_id: expect.any(Number),
```
The comment id should really match the one that was requested

Reply:
reviewed with Rayhaan via nchelp - ok to merge for the time-being